simulator-ui/src/main/resources/static/src/assets/css/filter-section.css

```diff
@@ -0,0 +1,174 @@
+.first-input-element {
+    border-top-left-radius: 0px;
+    border-bottom-left-radius: 0px;
+    border-left: 0;
+}
+.btn:active, .btn.active {
+    text-decoration: underline;
+    outline: 0;
+    background-image: none;
+    box-shadow: none;
+}
+
+.btn-primary,
+.btn-primary:focus {
+    background-color: #33abde !important;
+    border-color: #33abde !important;
+}
+
+.btn-primary:hover, .btn-primary.active:hover {
+    background-color: #33abde !important;
+    border-color: #33abde !important;
+}
+
+.btn-primary.active,
+.btn-primary.active:focus {
+    background-color: #0096D6 !important;
+    border-color: #0096D6 !important;
+}
+
+.btn-return {
+    color: #ffffff;
+    border-color: #d66e5b;
+    background: #d66e5b;
+}
+
+.btn-return:hover {
+    border-color: #d6857a;
+    background-color: #d6857a;
+}
+
+.filter-input:focus, .filter-input:active:focus, .filter-input.active:focus, .filter-input.focus, .filter-input:active.focus, .filter-input.active.focus {
+    outline: 0;
+    box-shadow: none !important;
+}
+
+.filter {
+    border-radius: 0 !important;
+    height: 100% !important;
+}
+
+.date-section {
+    display:flex;
+    justify-content:space-between;
+    horiz-align:center;
+}
+
+.date-box {
+    display:grid;
+    flex-grow:1;
+}
+
+.date-box + .date-box {
+    border-left:solid 20px #fff;
+}
+
+::ng-deep .mat-form-field-appearance-fill .mat-form-field-flex {
+    padding-top:0!important;
+    background-color:#fff!important;
+    align-items:center;
+}
+
+::ng-deep .mat-form-field-appearance-fill .mat-form-field-infix {
+    padding:0;
+    border:none;
+}
+
+::ng-deep .mat-form-field-wrapper {
+    margin-bottom:-10px;
+}
+
+::ng-deep .ngx-timepicker {
+    height:70%!important;
+    border-bottom:0!important;
+    padding:0;
+}
+
+::ng-deep input.mat-input-element {
+    margin-left:13px;
+    color:#69666b;
+}
+
+::ng-deep .ngx-material-timepicker-toggle {
+    padding:0 0 5px!important;
+    margin-right:20px;
+    filter:invert(37%) sepia(5%) saturate(6656%) hue-rotate(156deg) brightness(97%) contrast(77%);
+}
+
+::ng-deep.mat-form-field-ripple {
+    background-color:#000!important;
+}
+
+::ng-deep .mat-calendar-body-cell:not(.mat-calendar-body-disabled):hover > .mat-calendar-body-cell-content:not(.mat-calendar-body-selected):not(.mat-calendar-body-comparison-identical) {
+    background-color:#5bc0de;
+}
+
+::ng-deep .mat-calendar-body-today:not(.mat-calendar-body-selected):not(.mat-calendar-body-comparison-identical) {
+    border-color:#31708f;
+    background-color:#5bc0de!important;
```

Comment:
this file looks unformatted. as mentioned [here](https://github.com/citrusframework/citrus-simulator/pull/111#pullrequestreview-824913773), try avoiding `!important`..